### PR TITLE
Release note for new Apama-ctrl-mt-4c-16g microservice in 10.17 (PAB-3422)

### PR DESCRIPTION
**Pull request by kailol-sag**

### CODE REVIEW

File: content/release-10-17-0/streaming-analytics-10-17-0-bundle/10_17_0.md [hunk 1]
```diff
@@ -10,6 +10,13 @@ This release of Cumulocity IoT Streaming Analytics includes the Apama version 10
 See also [What's New In Apama 10.15.2](https://documentation.softwareag.com/pam/10.15.2/en/webhelp/pam-webhelp/index.html#page/pam-webhelp%2Fco-WhaNewInApa_10152_top.html)
 in the Apama documentation.
 
+### New multi-tenancy microservice
+
+The new Apama-ctrl-mt-4c-16g microservice supports multi-tenancy, but has limitations in this version.
+Currently, it only supports EPL apps and smart rules and does not support Analytics Builder.
+Furthermore, EPL apps are only enabled on the Management and Enterprise tenants, but disabled on the subtenants.
```
Comment (skom-sag) [inline content/release-10-17-0/streaming-analytics-10-17-0-bundle/10_17_0.md:17]: can you reword this to something that conveys a meaning like ... EPL apps are only enabled on the tenant that owns the microservice.

Comment (kailol-sag) [reply content/release-10-17-0/streaming-analytics-10-17-0-bundle/10_17_0.md]: ok, done here and also in c8y-docs.

**Review by skom-sag — APPROVED** [review]
